rpi_backlight/__init__.py

```diff
@@ -161,12 +198,22 @@ def power(self) -> bool:
         :type: bool
         """
         # 0 is on, 1 is off
-        return not self._get_value("bl_power")
+        if self._board_type == BoardType.RASPBERRY_PI:
+
+            return not self._get_value("bl_power")
+        elif self._board_type == BoardType.TINKER_BOARD:
+            if self._get_value("tinker_mcu_bl") == 0:
+                return 0
+            else:
+                return 1
```

Comment:
i think this is what we want

```suggestion
            return bool(self._get_value("tinker_mcu_bl"))
```

which will return `False` for 0 and `True` for every other number